docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch nodes with a user-specified label.
+* Custom SSL Profiles can be loaded from Kubernetes Secrets via Ingress TLS.
```

Comment:
Are we loading (i.e., creating) these on the BIG-IP? If so, how about:
Creation of BIG-IP SSLProfiles from Kubernetes secrets via Ingress TLS.